data/vanshiz.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Vanshika Bhargava",
+  "bio": "A sophomore student studying Btech in Computer Science"
```

Please add the missing comma here. 
```suggestion
  "bio": "A sophomore student studying Btech in Computer Science",
```